apps/web/src/pages/Pools/PoolAdvertisementPage/components/LanguageRequirementDialog.tsx

```diff
@@ -0,0 +1,461 @@
+          })}
+        </Dialog.Header>
+        <Dialog.Body>
+          <div>
+            <Heading
+              level={headingLevel}
+              size="h6"
+              data-h2-margin-top="base(0)"
+            >
+              {intl.formatMessage({
+                defaultMessage: "Language requirements on GC Digital Talent",
+                id: "h1UJ1Q",
+                description: "Sub-heading for language requirements dialog",
+              })}
+            </Heading>
+            <p>
+              {intl.formatMessage({
+                defaultMessage:
+                  "These are the language profiles that you can find on GC Digital Talent",
+                id: "sry9Wl",
+                description:
+                  "Language profiles list for language requirements dialog",
+              }) + intl.formatMessage(commonMessages.dividingColon)}
+            </p>
+
+            <DescriptionList.Root>
+              <DescriptionList.Item
+                title={intl.formatMessage(languageProfiles.englishEssential)}
+              >
+                {intl.formatMessage(
+                  languageProfileDescriptions.englishEssential,
+                )}
+              </DescriptionList.Item>
+              <DescriptionList.Item
+                title={intl.formatMessage(languageProfiles.frenchEssential)}
+              >
+                {intl.formatMessage(
+                  languageProfileDescriptions.frenchEssential,
+                )}
+              </DescriptionList.Item>
+              <DescriptionList.Item
+                title={intl.formatMessage(
+                  languageProfiles.bilingualIntermediate,
+                )}
+              >
+                {intl.formatMessage(
+                  languageProfileDescriptions.bilingualIntermediate,
+                )}
+              </DescriptionList.Item>
+              <DescriptionList.Item
+                title={intl.formatMessage(languageProfiles.bilingualAdvanced)}
+              >
+                {intl.formatMessage(
+                  languageProfileDescriptions.bilingualAdvanced,
+                )}
+              </DescriptionList.Item>
+              <DescriptionList.Item
+                title={intl.formatMessage(languageProfiles.various)}
+              >
+                {intl.formatMessage(languageProfileDescriptions.various)}
+              </DescriptionList.Item>
+            </DescriptionList.Root>
+          </div>
+          <div data-h2-margin-bottom="base(x1)">
+            <Heading
+              level={headingLevel}
+              size="h6"
+              data-h2-margin-top="base(0)"
+            >
+              {intl.formatMessage({
+                defaultMessage: "Bilingual positions",
+                id: "i9ovz7",
+                description: "Sub-heading for language requirements dialog",
+              })}
+            </Heading>
+            <div data-h2-display="base(grid)" data-h2-gap="base(x.5)">
```

Comment:
This `div` could be avoided if we just use `data-h2-margin="base(x.5 0)"` on the `<p>` tags.

Reply:
Fixed by [7957bec](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10628/commits/7957bec46d036ea078394010540617660e66b9df)